scripts/backend.ps1

```diff
@@ -93,6 +87,18 @@ class HypervInstance : Instance {
     [void] AttachVMDvdDrive ($DvdDrive) {
         $this.Backend.AttachVMDvdDrive($this.Name, $DvdDrive)
     }
+
+    [Array] GetVMHDD () {
```

Comment:
GetVMDisk -- to be consistent with the naming of Hyper-V
I suggest GetDisk, as this is already an instance method